Cargo.lock

```diff
@@ -93,6 +94,26 @@ dependencies = [
 ]
 
 [[package]]
+name = "errno"
```

Comment:
Oh right, duh! Forgot this was a dependency of `file-lock`.

That being said I believe that this crate doesn't provide anything beyond `io::Error::last_os_error()` and `io::Error::raw_os_error()`, so I'd prefer if the `file-lock` crate were adjusted to not have this dependency.